acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/proxy/PassThroughAcceptanceTest.java

```diff
@@ -33,24 +33,33 @@
 
   @Test
   public void ethGetTransactionCountReturnCorrectNumber() {
-    final BigInteger startTransactionCount =
+    final BigInteger pantheonTransactionCount =
         ethNode().transactions().count(richBenefactor().address());
-    submitTransactionAndWaitForBlock();
+    BigInteger ethSignerTransactionCount =
```

Comment:
final

Reply:
Can't be final, because I'm reusing this variable ...